doc/user/ruby-managers.md

```diff
@@ -0,0 +1,28 @@
+# Ensuring Ruby managers are configured correctly
+
+If you are using a Ruby manager like `rvm`, `rbnev`, or `chruby` make 
+sure that the manager does not set environment variables `GEM_HOME`, `GEM_PATH`,
+and `GEM_ROOT`. The variables are picked up by truffleruby (as any other Ruby 
+implementation would do) causing truffleruby to pickup wrong gem-home directory 
+instead of it's own.
+
+It can be easily fixed by clearing the environment with one of the following 
+commands:
+
+```bash
+rvm use system
+rbenv system
+chruby system
+```
+
+Otherwise, unset the variables with:
+
+```bash
+unset GEM_HOME GEM_PATH GEM_ROOT
+```
+
+It will be easier when TruffleRuby can be added to ruby managers, we are working 
+on it.  
+
+Next step: [Installing gems](https://github.com/graalvm/truffleruby/blob/master/doc/user/isntalling-gems.md).
```

Comment:
All links can be relative, GitHub handles it nicely and it means it works on branches (test it with the markdown preview in the PR).
BTW, typo in the link here.